userdocs/src/usage/minimum-iam-policies.md

```diff
@@ -40,8 +40,6 @@ AmazonEC2FullAccess
                         "autoscaling.amazonaws.com",
                         "ec2scheduled.amazonaws.com",
                         "elasticloadbalancing.amazonaws.com",
-                        "eks.amazonaws.com",
```

Comment:
Thanks for fixing this. Can you please also add a note on line 8 that makes it clear that this is the managed policy?

Reply:
Hi, I updated about your point.
Could you please check this ?
Please let me know if you have any problems.

Reply:
Hi, I updated about your point.
Could you please check this ?
Please let me know if you have any concerns.